frontend/src/main/pages/HelpRequest/HelpRequestIndexPage.js

```diff
@@ -1,17 +1,47 @@
 import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import { hasRole, useCurrentUser } from "../../utils/currentUser";
```

Comment:
I wonder if this should be an absolute path instead of a relative path.  We typically try to avoid relative paths in our code bases.